app/lib/LevelLoader.coffee

```diff
@@ -3,6 +3,7 @@ LevelComponent = require 'models/LevelComponent'
 LevelSystem = require 'models/LevelSystem'
 Article = require 'models/Article'
 LevelSession = require 'models/LevelSession'
+{me} = require 'core/auth'
```

Comment:
It looks like I am adding a dependency. But code in this file already references `me`. Just implicitly.